packages/api/package.json

```diff
@@ -99,20 +99,6 @@
     "xml2js": "^0.4.22"
   },
   "devDependencies": {
-    "@cumulus/test-data": "1.22.1",
-    "ava": "^3.8.2",
-    "babel-loader": "^8.0.6",
-    "babel-plugin-source-map-support": "^2.1.1",
-    "babel-preset-env": "^1.7.0",
-    "copy-webpack-plugin": "^4.6.0",
-    "fs-extra": "^7.0.0",
-    "nock": "^10.0.0",
-    "nyc": "^14.0.0",
-    "proxyquire": "^2.0.0",
-    "rewire": "^4.0.1",
-    "sinon": "^7.1.1",
-    "supertest": "^3.3.0",
-    "webpack": "~4.5.0",
-    "webpack-cli": "~2.0.14"
+    "@cumulus/test-data": "1.22.1"
```

Comment:
I notice that this dependency (and a couple of others in other places) are left behind in non-root `devDependencies`. Just curious as to why that is.